angular-client/src/services/theme.service.ts

```diff
@@ -7,9 +7,9 @@ export default class Theme {
   static readonly textColor: string = 'color: #EFEFEF; ';
   static readonly textStyle: string = this.font + this.textColor;
   static readonly boldedText: string = this.textStyle + 'font-weight: bold; ';
-  static readonly header: string = this.boldedText + 'fontSize: 20px;';
+  static readonly header: string = this.boldedText + 'font-size: 20px; margin: 0';
```

Comment:
you'll need to check that everywhere else this variant is used still looks fine (specifically for header and infoUnit). I would make new variants if these are used in a bunch of places